### PR TITLE
fix: issue for db-button hover and active states for other tags like <a>

### PR DESCRIPTION
**Pull request by nmerget**
## Proposed changes

- hover and active did not work for `<a class="db-button">`

## Types of changes

-   [x] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Refactoring (fix on existing components or architectural decisions)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: __snapshots__/button/component/firefox/DBButton-should-only-have-icon-for-variant-text.png
Comment (mfranzke) [inline __snapshots__/button/component/firefox/DBButton-should-only-have-icon-for-variant-text.png:1]: is the change to a dark background-color intended on this one?

File: __snapshots__/button/component/firefox/DBButton-should-only-have-icon-for-variant-outlined.png
Comment (mfranzke) [inline __snapshots__/button/component/firefox/DBButton-should-only-have-icon-for-variant-outlined.png:1]: @nmerget is the change to a dark background-color done intentionally on this one? Or was the previous screenshot just outdated ?

File: __snapshots__/navigation-item/showcase/mobile-safari/regular/neutral/DBNavigationItem-should-match-screenshot.png
Comment (mfranzke) [inline __snapshots__/navigation-item/showcase/mobile-safari/regular/neutral/DBNavigationItem-should-match-screenshot.png:1]: @nmerget positioning the arrow on the full width navigation-item looks intentionally, but is it ?

File: __snapshots__/textarea/showcase/mobile-safari/regular/neutral/DBTextarea-should-match-screenshot.png
Comment (mfranzke) [inline __snapshots__/textarea/showcase/mobile-safari/regular/neutral/DBTextarea-should-match-screenshot.png:1]: @nmerget it looks like that the arrows for vertical scrolling got to appear slightly smaller; has this change been made intentionally ?

File: __snapshots__/textarea/showcase/webkit/regular/neutral/DBTextarea-should-match-screenshot.png
Comment (mfranzke) [inline __snapshots__/textarea/showcase/webkit/regular/neutral/DBTextarea-should-match-screenshot.png:1]: @nmerget it looks like that the arrows for vertical scrolling got to appear slightly smaller; has this change been made intentionally ?

**Review by mfranzke — APPROVED** [review]

File: packages/foundations/scss/colors/_variants.scss [hunk 1]
```diff
@@ -39,7 +39,7 @@
 }
 
 @mixin bg-transparent-hover() {
-	@extend %bg-transparent;
+	@include get-variant-bg-color(0);
```
Comment (mfranzke) [inline packages/foundations/scss/colors/_variants.scss:42]: @nmerget `%bg-transparent` still exists in further places – doesn't this need to get replaced in all?

Comment (nmerget) [reply packages/foundations/scss/colors/_variants.scss:42]: I think it is fine there, the problem was just a placeholder inside a mixin for firefox. The other components use the placeholder directly

**Review by mfranzke — APPROVED** [review]

**Pull request merged**